### ISSUE TITLE
Part for OR gate

### ISSUE THREAD

**Issue by preynoldssco** [PRNicovich/NicoLase#2]
Hi! I can't really find a part here in the UK for the OR gate you have listed on the BOM for the PCB. I'm just getting the bits together to build it...

This seems close?
http://www.mouser.co.uk/ProductDetail/Diodes-Incorporated/74AHC1G32SE-7/?qs=aOs975IaWlmmUQ8iQjImmQ==

If you have time could you add a link/part number/supplier to the BOM?

**Comment by PRNicovich** [comment]
Added the part number for what I used (74HC1G32GW from NXP).  This is the same part available from Mouser:

http://www.mouser.co.uk/ProductDetail/NXP-Semiconductors/74HC1G32GW125/?qs=sGAEpiMZZMtMa9lbYwD6ZKUVY3zpSJbe2QB%252bxb%252bVeNA%3d

I think the one that you linked draws a bit more current than mine, but that's a difference of microamps.  I can't find a reason your part would not work just as well as mine here.
